Patches/Androids Expanded/AlienRace/AlienRace_Droids.xml

```diff
@@ -0,0 +1,246 @@
+				  <xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="ChjSpacerBattleDroid"]/statBases</xpath>
+				  <value>
+					<MeleeCritChance>1.4</MeleeCritChance>
+					<MeleeParryChance>1.5</MeleeParryChance>
+					<SmokeSensitivity>0</SmokeSensitivity>
+				  </value>
+				</li>
+				<!--Recon Armor Equivalent-->
+				<li Class="PatchOperationReplace">
+				  <xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="ChjSpacerBattleDroid"]/statBases/ArmorRating_Sharp</xpath>
+				  <value>
+				    <ArmorRating_Sharp>16</ArmorRating_Sharp>
+				  </value>
+				</li>
+				
+				<li Class="PatchOperationReplace">
+				  <xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="ChjSpacerBattleDroid"]/statBases/ArmorRating_Blunt</xpath>
+				  <value>
+				    <ArmorRating_Blunt>36</ArmorRating_Blunt>
+				  </value>
+				</li>
+				
+				<li Class="PatchOperationReplace">
+				  <xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="ChjSpacerBattleDroid"]/tools</xpath>
+				  <value>
+				    <tools>
+					  <li Class="CombatExtended.ToolCE">
+						<label>left razor-fist</label>
+						<capacities>
+						  <li>Cut</li>
+						  <li>Stab</li>
+						  <li>Blunt</li>
+						</capacities>
+						<power>27</power>
+						<cooldownTime>1.76</cooldownTime>
+						<linkedBodyPartsGroup>LeftDroidHand</linkedBodyPartsGroup>
+						<armorPenetrationBlunt>60</armorPenetrationBlunt>
+						<armorPenetrationSharp>24</armorPenetrationSharp>
+					  </li>
+					  <li Class="CombatExtended.ToolCE">
+						<label>right razor-fist</label>
+						<capacities>
+						  <li>Cut</li>
+						  <li>Stab</li>
+						  <li>Blunt</li>
+						</capacities>
+						<power>27</power>
+						<cooldownTime>1.76</cooldownTime>
+						<linkedBodyPartsGroup>RightDroidHand</linkedBodyPartsGroup>
+						<armorPenetrationBlunt>60</armorPenetrationBlunt>
+						<armorPenetrationSharp>24</armorPenetrationSharp>
+					  </li>
+					  <li Class="CombatExtended.ToolCE">
+						<label>lower left razor-fist</label>
+						<capacities>
+						  <li>Cut</li>
+						  <li>Stab</li>
+						  <li>Blunt</li>
+						</capacities>
+						<power>27</power>
+						<cooldownTime>1.76</cooldownTime>
+						<linkedBodyPartsGroup>LeftDroidHand</linkedBodyPartsGroup>
+						<armorPenetrationBlunt>60</armorPenetrationBlunt>
+						<armorPenetrationSharp>24</armorPenetrationSharp>
+					  </li>
+					  <li Class="CombatExtended.ToolCE">
+						<label>lower right razor-fist</label>
+						<capacities>
+						  <li>Cut</li>
+						  <li>Stab</li>
+						  <li>Blunt</li>
+						</capacities>
+						<power>27</power>
+						<cooldownTime>1.76</cooldownTime>
+						<linkedBodyPartsGroup>LowerRightDroidHand</linkedBodyPartsGroup>
+						<armorPenetrationBlunt>60</armorPenetrationBlunt>
+						<armorPenetrationSharp>24</armorPenetrationSharp>
+					  </li>
+					  <li Class="CombatExtended.ToolCE">
+						<label>giant body slam</label>
+						<capacities>
+						  <li>Blunt</li>
+						</capacities>
+						<power>65</power>

```

Comment:
The attack power--and even the concept of the attack--is just kind of ridiculous here. "Body slam"? The entire battle droid is doing what? Leaping into the air and dropping the People's Elbow on its target? Or picking the target up (whatever it may be, like an elephant or centipede) and pile driving it? 

Just replace this with a standard HeadAttackTool headbutt or something, please. Probably shouldn't exceed the overall effectiveness of the Centipede in regards to power and blunt AP.